CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.15.0] - 2023-05-16
+
+### Changed
+
+- Cache implementation now uses rocksdb instead of microkelvin
```

Comment:
Add reference to the issue you're resolving, and add the link at the end of the changelog (follow the same structure of the other entries)